.github/workflows/publish-to-hex.yml

```diff
@@ -1,17 +0,0 @@
-on:
-  push:
-    tags:
-      - '*'
```

Comment:
This was added by @gilbertwong96 who wanted to publish it to hex once upon a time. Josef and me never bothered. @erszcz I think you can do this in your prefered way. (Personally I never remember how to publish stuff to hex.)

Perhaps we can just change this to take action only when a tag is created on master? If it works, it seems convenient...

Reply:
The tool I found to generate the changelog requires a tag present in the remote repo. But this hook automatically triggers publishing a package when such a tag is pushed. That's why I think doing this manually is fine. It's a task to do once in a few months, I think, so not a lot of overhead.

Reply:
Thx, very good.

Reply:
> The tool I found to generate the changelog requires a tag present in the remote repo. But this hook automatically triggers publishing a package when such a tag is pushed. That's why I think doing this manually is fine. It's a task to do once in a few months, I think, so not a lot of overhead.

Which tool did you use to generate the changelog? What about generating a changelog automatically with the GitHub workflow action?

Reply:
@gilbertwong96

> Which tool did you use to generate the changelog?

[`github-changelog`](https://github.com/cfpb/github-changelog) as I wrote in the changelog itself. There was also a bit of manual editing involved.

In general, neither the PR titles, nor the commit messages in the repo are formatted consistently enough - it's rather a free-for-all. This makes generating the changelog completely automatically quite hard - I did try with [`git-chglog`](https://github.com/git-chglog/git-chglog) first, but the result was unsatisfactory.

Anyway, I don't think the overhead of doing this once in a couple of months is worth the effort of automation. Unless somebody wants to step forward with the initiative and propose a plan how to tackle it in the future - I'd be happy to see it happening :)